src/main/inAppNotifications/graphql/InAppNotificationsFragments.graphql

```diff
@@ -0,0 +1,95 @@
+fragment InAppNotificationCalloutPublished on InAppNotificationCalloutPublished {
+  callout {
+    id
+    type
+    framing {
+      id
+      profile {
+        id
+        displayName,
+        url,
+        visual(type: CARD) {
+          ...VisualUri
+        }
+      }
+    }
+  }
+  space {
+    id
+    level
+    profile {
+      id
+      displayName,
+      url,
+      visual(type: CARD) {
+        ...VisualUri
+      }
+    }
+  }
+  triggeredBy {
+    id
+    profile {
+      id
+      displayName,
+      url,
+      visual(type: AVATAR) {
+        ...VisualUri
+      }
+    }
+  }
+}
+
+fragment InAppNotificationCommunityNewMember on InAppNotificationCommunityNewMember {
+  triggeredBy {
+    id
+    profile {
+      id
+      displayName,
+      url,
+      visual(type: AVATAR) {
+        ...VisualUri
+      }
+    }
+  }
+  space {
+    id
+    level
+    profile {
+      id
+      displayName,
+      url,
+      visual(type: CARD) {
+        ...VisualUri
+      }
+    }
+  }
+  actor {
+    id
+    __typename
```

Comment:
add a comment smiliar to the one in `src/domain/journey/space/pages/SpaceAccount/SpaceAccount.graphql` please, in theory it should always be there but wanted to keep track of the `__typename`s that we actually use